### ISSUE TITLE
Error using the sample code

### ISSUE THREAD

**Issue by shawnchapla-8451** [CamiloTerevinto/TerevintoSoftware.PythonPkceClient#3]
Hello.  I've just come across this and wanted to test it out to see if it could work for my project, using the sample code provided in the README.  I'm getting an error generated when it executes the line ```pkce_token = login_client.login()```:

```TypeError: PkceToken.__init__() missing 3 required positional arguments: 'token_type', 'expires_in', and 'scope'```

It does properly redirect to the IdP login page and back again to the code before throwing the error. I just modified the example's PkceLoginConfig values with those for our environment, and haven't made any other changes:

```
config = PkceLoginConfig(
    authorization_uri="<IdP auth URL>",
    token_uri="<IdP token URL>",
    scopes=[ "openid", "profile", "<custom scope>", "offline_access" ],
    client_id="<client ID>",
    internal_port=9000,
    add_random_state=True,
    random_state_length=32,
    verify_authorization_server_https=False
)
```
Is there something basic I've missed?  I'm using Python v3.10.12.

Thanks in advance.

**Comment by CamiloTerevinto** [comment]
Hi @shawnchapla-8451, what IdP are you using? The arguments missing tell me that the token is missing basic information, which typically happens if you are not getting a success response from the IdP. It could also be that the IdP doesn't return that information with the token, though...

**Comment by shawnchapla-8451** [comment]
> Hi @shawnchapla-8451, what IdP are you using? The arguments missing tell me that the token is missing basic information, which typically happens if you are not getting a success response from the IdP. It could also be that the IdP doesn't return that information with the token, though...

Thanks for your response.  I'm testing it with Okta as the IdP.  I'd have to check with the team that manages the Okta implementation to see if these are customizations they've made, but an access token I can generate with a tool like Insomnia shows these as the likely claim names Okta is using:

- "scp" instead of "scope"
- "exp" instead of "expires_in"

I'm not sure what claim is standing in for "token_type".  Is there a way to tell  PkceLoginConfig about implementation-specific claim names?

**Comment by CamiloTerevinto** [comment]
Hey @shawnchapla-8451, I've released version 0.1.0 just now which gives more flexibility for this through the `TokenConfigMap` class. Can you give that a try and let me know how it works for you?
